src/main/java/com/google/sps/servlets/CaptionService.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class CaptionService {
```

Comment:
Is there a reason for having Caption and CaptionService as separate classes? Just wondering!

Reply:
I can relate it is annoying to have two classes instead of one 😅 The reason is to avoid actually making Internet connections/API calls when testing.